ConsoleFileManager 2.0/Commands/ChangeDirectoryCommand.cs

```diff
@@ -16,7 +17,7 @@ public ChangeDirectoryCommand(IUserInterface UserInterface, FileManagerLogic Fil
 
     public override void Execute(string[] args)
     {
-        if (args.Length != 2 || string.IsNullOrWhiteSpace(args[1]))
+        if (args.Length < 2 || string.IsNullOrWhiteSpace(args[1]))
```

Comment:
```suggestion
        if (args.Length < 2 || args[1] is not { Length: > 0 })
```

Reply:
Называется "сопоставление с образцом".

Reply:
```suggestion
        if (args is not [_, { Length: > 0 } ])
```
Если включить экспериментальные функции языка (в файле `.csproj` в раздел `PropertyGroup` добавить `<LangVersion>preview</LangVersion>`), то можно использовать более продвинутый вариант сопоставления с образцом для списков. Он станет доступен осенью (в октябре) с выходом `.NET7`